plutus-playground-client/src/Editor/View.purs

```diff
@@ -118,11 +137,11 @@ editorFeedback editorState@(State { currentCodeIsCompiled, feedbackPaneMinimised
         ]
     ]
   where
-  feedbackPaneClasses =
-    if feedbackPaneMinimised then
-      [ ClassName "editor-feedback", ClassName "minimised" ]
-    else
-      [ ClassName "editor-feedback" ]
+  feedbackPaneClasses = case feedbackPaneMinimised, feedbackPaneExtend of
+    false, 0 -> [ ClassName "editor-feedback" ]
+    true, 0 -> [ ClassName "editor-feedback", ClassName "minimised" ]
+    false, size -> [ ClassName "editor-feedback", ClassName $ "expanded-" <> show size ]
+    true, size -> [ ClassName "editor-feedback", ClassName "minimised", ClassName $ "expanded-" <> show size ]
```

Comment:
`editor-feedback` is always used, right? So how about `[ ClassName "editor-feedback", case ... ]`?

Reply:
Ah yes, almost. But in one case it's the only class, so `[ ClassName "editor-feedback" ] <> case ...` ought to do it.